samples/extensions/host_image_copy/host_image_copy.cpp

```diff
@@ -100,7 +100,7 @@ void HostImageCopy::load_texture()
 	VkFormatProperties2 format_properties_2{};
 	format_properties_2.sType = VK_STRUCTURE_TYPE_FORMAT_PROPERTIES_2;
 	format_properties_2.pNext = &format_properties_3;
-	vkGetPhysicalDeviceFormatProperties2(get_device().get_gpu().get_handle(), image_format, &format_properties_2);
```

Comment:
Can you move this into a separate PR? Would make the review process a lot easier and (at least for this change) also faster.

Reply:
Agreed.  Reverted in https://github.com/KhronosGroup/Vulkan-Samples/pull/1173/commits/ae5e98d0389a2ce85180d48a2ee630e416a143d1.  I will raise a new PR.